build.sbt

```diff
@@ -31,7 +31,21 @@ lazy val formattingSettings = SbtScalariform.scalariformSettings ++ Seq(
       .setPreference(PlaceScaladocAsterisksBeneathSecondAsterisk, true)
 )
 
+lazy val scoverageSettings = Seq(
```

Comment:
Good catch. Removing them as they're not needed (yet). It'd be better to include them when they're required.